osi_hostvehicledata.proto

```diff
@@ -2,31 +2,356 @@ syntax = "proto2";
 
 option optimize_for = SPEED;
 
+import "osi_version.proto";
 import "osi_common.proto";
+import "osi_object.proto";
 
 package osi3;
 
-//
-// \brief Interface for host vehicle data that is available to sensors and
-// other functions due to host vehicle's internal communication.
+// \brief Host vehicle data is about the perception of the vehicle about it's own, internal states.
+// It describes data that the host vehicle knows about itself,
+// e.g. from location sensors, internal sensors, board net etc.
+// A dynamic model can serve as input provider.
+// Sensors, mockups or other modules can make usage of the host vehicle data.
+// It consists of different messages categorizing the vehicle in:
+// Vehicle-Basics, Vehicle-Powermanagement, Vehicle-Powertrain, Vehicle-SteeringWheel, Vehicle-Wheels, Vehicle-Localization.
 //
 // \image html OSI_HostVehicle.svg
 //
-// All coordinates and orientations are relative to the global ground truth
-// coordinate system.
+// All coordinates and orientations are relative to the global ground truth coordinate system.
+// Otherwise it is mentioned explicitly.
 //
 message HostVehicleData
 {
+    // Deprecated: Will be removed in next major release. Moved to VehiclePositionAndKinematics.
     // Current estimated location based on GPS- and related navigation sensors.
     //
     // \note Note that dimension and base_polygon need not be set.
     //
     optional BaseMoving location = 1;
 
-    // Current estimated location error based on GPS- and related navigation
+    // Deprecated: Will be removed in next major release. Moved to VehiclePositionAndKinematics.
+    // Current estimated location error based on GPS- and related navigation 
     // sensors.
     //
     // \note Note that dimension and base_polygon need not be set.
     //
     optional BaseMoving location_rmse = 2;
+
```

Comment:
We might want to add fields for the current time of day, as known to the vehicle, and a simulation-time current timestamp to indicate for which point in simulation time the fields of this message are considered valid (which indicates the time when the car networks would have transmitted this data, which is different from individual measurement times that went into the calculations for the values).

Reply:
I also agree that we need an addtional timestamp that excludes the additional latency from the system (this is the time of the actual measurement when it was measured (real data) or generated (simulation)).